Rakefile

```diff
@@ -290,6 +290,10 @@ task :install => :build do
   if CERTS_SCENARIOS.any?
     mkdir_p "#{DATADIR}/foreman-installer/katello-certs/scenarios.d"
     cp_r 'katello_certs/hooks', "#{DATADIR}/foreman-installer/katello-certs"
+    # 20-certs_update.rb lives in katello, but is *also* required in katello-certs
```

Comment:
I'm leaning to copying the files in git so there's a better overview there. We already copied other hooks.

Reply:
Did we? I didn't find any duplicate files (only names)

Reply:
I guess I only copied relevant sections then. My worry is that nobody looks at the Rakefile and will have issues tracking down where a file came from. A relative symlink would also work and be easier to track down in development.

Reply:
Oh yeah, a symlink would work

Reply:
updated to be symlinks